app/controllers/thesis_controller.rb

```diff
@@ -23,6 +23,12 @@ def show
     @thesis = Thesis.find(params[:id])
   end
 
+  # Do not name this simply 'process' or you will shadow a built-in controller
+  # function and then you will be sad.
+  def process_theses
+    @theses = Thesis.all.order('grad_date ASC').page(params[:page]).per(25)
```

Comment:
I think `Thesis.order` is the same as `Thesis.all.order` and is more 🌈

Reply:
++